docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -2,142 +2,366 @@
+
+   {{%expand "Click here for details." %}}
+   For example, follow the same steps as the [Quick Start](https://oracle.github.io/weblogic-kubernetes-operator/quickstart/) guide up through the [Prepare for a domain]({{< relref "/quickstart/prepare.md" >}}) step.
+
+   > **Note:** Skip the Quick Start steps for obtaining a WebLogic image because you will be creating your own Docker image.
+   {{% /expand%}}
+
+1. _Make sure there are no conflicting WebLogic related domains, Ingresses, secrets, or config maps that are already deployed to namespace `sample-domain1-ns`._
+
+   {{%expand "Click here for details." %}}
+
+   If any of the following commands reveal WebLogic related resources, then the simplest way to avoid conflicts is to delete them using the corresponding `kubectl delete TYPE NAME` verb:
 
    ```
-   cd <location of empty temporary directory with 10g of space>
-   export WORKDIR=$(pwd)
+   kubectl get domains    -n sample-domain1-ns
+   kubectl get pods       -n sample-domain1-ns
+   kubectl get configmaps -n sample-domain1-ns
+   kubectl get secrets    -n sample-domain1-ns
+   kubectl get ingresses  -n sample-domain1-ns
    ```
 
-    If you do not set this environment variable, then the sample scripts will default to `/tmp/$USER/model-in-image-sample-work-dir` where `$USER` is your user name.
+   For example, if the `get domains` command reveals domain `mydomain`, then call `kubectl delete domain mydomain -n sample-domain1-ns` to shut it down.
+
+   > **WARNING:** If you delete an existing domain, then wait for its pods to shut down. To monitor the sample domain namespace as pods shut down, use `kubectl get pods -n sample-domain1-ns -w` and type `ctrl-c` to exit.
+   {{% /expand%}}
+
+
+1. _Set up Ingresses that will redirect HTTP from Traefik port 30305 to the clusters in this sample's WebLogic domains._
 
-5. Deploy the operator and set up the operator to manage the namespace, `sample-domain1-ns`. Optionally, deploy a Traefik load balancer that manages the same namespace. For example, follow the same steps as the [Quick Start](https://oracle.github.io/weblogic-kubernetes-operator/quickstart/), up through the [Prepare for a domain]({{< relref "/quickstart/prepare.md" >}}) step.
+    Run one of the following commands:
 
-   Note that:
-   - Deploying the Traefik load balancer is optional, but is a prerequisite for testing the web application that's deployed to WebLogic as part of this sample.
-   - You can skip the Quick Start steps for obtaining a WebLogic image because you will be creating your own Docker image.
+    - To deploy the Ingresses, one option is to use your favorite editor to cut and paste the following YAML to a file called `/tmp/mii-sample/ingresses/myingresses.yaml` and then call `kubectl apply -f /tmp/mii-sample/ingresses/myingresses.yaml`:
```

Comment:
i found this a bit magical... i think it needs to be explained more.  maybe move the note from 270 up to above this and expand it a little bit?   so i know before i get to this step what i am about to do, otherwise i am left wondering for a little while and might go off on a tangent trying to work it out only to come back later and find the answer on the next line :(